src/main/java/uk/gov/ons/ctp/response/casesvc/client/CollectionExerciseSvcClient.java

```diff
@@ -1,10 +1,12 @@
 package uk.gov.ons.ctp.response.casesvc.client;
 
+import static net.logstash.logback.argument.StructuredArguments.kv;
```

Comment:
Needed to provide the key-value structured logging now we've moved to `slf4j` as godaddy previously provided this as part of its implementation.